app/models/annotations/status.rb

```diff
@@ -105,6 +105,10 @@ def destroy
 
   private
 
+  def set_annotator
+    self.annotator = User.current unless User.current.nil?
+  end
+
```

Comment:
Override it for status only to set annotator with current user as base one not set annotator if it set and this behavior not working with revisions and always show creator on annotations logs https://mantis.meedan.com/view.php?id=5665#c19477